pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "alpaca-py"
-version = "0.11.0"
+version = "0.11.1"
```

Comment:
not for this PR. Shall we try to consider using `poetry-dynamic-versioning`?

Reply:
Absolutely, let's discuss it on slack and I'll draft another PR for it